docs/_snippets/addon-vitest-run-tests.md

```diff
@@ -0,0 +1,11 @@
+```shell renderer="common" language="js" packageManager="npm"
+npm run test
+```
+
+```shell renderer="common" language="js" packageManager="pnpm"
+pnpm run test
+```
+
+```shell renderer="common" language="js" packageManager="yarn"
+yarn test
+```
```

Comment:
this should be `yarn/pnpm/npm exec vitest` @kylegach 

the add command doesn't add a `test` script

Reply:
I addressed this in the prose & snippets around this one.